### ISSUE TITLE
What is a well lr for train the voc datasets?

### ISSUE THREAD

**Issue by lingtengqiu** [jfzhang95/pytorch-deeplab-xception#21]
When I use U model to train voc. I find the lr = 1e-7 is to small .the model  convergence is TO SLOW .what is Ur good choice for learning rate ? how about 1e-4

**Comment by lingtengqiu** [comment]
Thanks for you kindly reply I am training on a modified version based on code , I want to try you newest deeplab_xception.py
